pep-0484.txt

```diff
@@ -313,9 +313,11 @@ holds a set of classes representing builtin types with generics, namely:
 * FrozenSet, used as ``FrozenSet[element_type]``
 
 * Tuple, used as ``Tuple[index0_type, index1_type, ...]``.
-  Arbitrary-length tuples might be expressed using ellipsis, in which
-  case the following arguments are considered the same type as the last
-  defined type on the tuple.
+  Arbitrary-length tuples are expressed using a final ellipsis, in which
+  case the type before the ellipsis may be repeated zero or more times.
```

Comment:
This should be stricter still -- Tuple[t1, t2, ...] should be disallowed. Only one type followed by an ellipsis should be acceptable.